docs/resources/policy_transit_gateway_static_route.md

```diff
@@ -54,9 +49,9 @@ The following arguments are supported:
 * `nsx_id` - (Optional) The NSX ID of this resource. If set, this ID will be used to create the resource.
 * `parent_path` - (Required) Path of parent object
 * `network` - (Required) Specify network address in CIDR format.
-* `next_hop` - (Required) List of Next Hops, each with those arguments:
+* `next_hop` - (Required) List of next hops (only one supported),each with those arguments:
```

Comment:
It's not a list anymore, just a nested object. Could be "Next hop, with those arguments"